src/lib/Scenes/Artwork/Components/__stories__/ArtworkTombstone.story.tsx

```diff
@@ -1,148 +1,44 @@
-                gravityID: "some-rando",
-                is_followed: false,
-              },
-            ],
-            cultural_maker: null,
-            dimensions: {
-              in: "15 × 20 in",
-              cm: "38.1 × 50.8 cm",
-            },
-            edition_of: "Edition 100/200",
-            attribution_class: {
-              short_description: "This is an edition of something",
-            },
-          } as any
         }
-      />
-    )
-  })
-  .add("Tombstone with 3 artists", () => {
-    return (
-      <ArtworkTombstone
-        artwork={
-          {
-            title: "Hello im a title",
-            medium: "Painting",
-            date: "1992",
-            artists: [
-              {
-                name: "Andy Warhol",
-                href: "/artist/andy-warhol",
-                id: "1234",
-                gravityID: "andy-warhol",
-                is_followed: false,
-              },
-              { name: "Alex Katz", href: "/artist/alex-katz", id: "4354", gravityID: "alex-katz", is_followed: false },
-              {
-                name: "Pablo Picasso",
-                href: "/artist/pablo-picasso",
-                id: "3433",
-                gravityID: "pablo-picasso",
-                is_followed: false,
-              },
-            ],
-            cultural_maker: null,
-            dimensions: {
-              in: "15 × 20 in",
-              cm: "38.1 × 50.8 cm",
-            },
-            edition_of: "Edition 100/200",
-            attribution_class: {
-              short_description: "This is an edition of something",
-            },
-          } as any
-        }
-      />
-    )
-  })
-  .add("tombstone with cultural maker", () => {
-    return (
-      <ArtworkTombstone
-        artwork={
-          {
-            title: "Hello im a title",
-            medium: "Painting",
-            date: "1992",
-            artists: [],
-            cultural_maker: "some guy idk",
-            dimensions: {
-              in: "15 × 20 in",
-              cm: "38.1 × 50.8 cm",
-            },
-            edition_of: "Edition 100/200",
-            attribution_class: {
-              short_description: "This is an edition of something",
-            },
-          } as any
-        }
-      />
-    )
-  })
+      `}
+    />
+  )
+}
+const artworkTombstoneArtwork = {
+  title: "Hello im a title",
+  medium: "Painting",
+  date: "1992",
+  artists: [
+    {
+      name: "Andy Warhol",
+      href: "/artist/andy-warhol",
+      id: "1234",
+      gravityID: "andy-warhol",
+      is_followed: false,
+      " $fragmentRefs": null,
+    },
+  ],
+  cultural_maker: null,
+  dimensions: { in: "15 × 20 in", cm: "38.1 × 50.8 cm" },
+  edition_of: "Edition 100/200",
+  attribution_class: { short_description: "This is an edition of something" },
+  " $refType": null,
+}
+storiesOf("Artwork/Components").add("New Component", () => {
+  return <MockArtworkTombstone artwork={artworkTombstoneArtwork} />
```

Comment:
@ds300 when you get in tomorrow... it would be ideal if we could use `MockRelayRenderer` here, but importing `MockRelayRenderer` seems to cause storybooks to not compile 😮. Wondering if you have any ideas here/would mind taking a look?

![image](https://user-images.githubusercontent.com/2081340/58836696-7ff32b00-8627-11e9-85bc-31e12e3a2a26.png)